test/JDBC/expected/BABEL-4926.out

```diff
@@ -0,0 +1,225 @@
+-- create a new table BABEL_4926 1
+create table BABEL_4926(babel_4926_COL int);
+go
+select babel_4926_COL from BABEL_4926;
+go
+~~START~~
+int
+~~END~~
+
+-- Create a new table BABEL_4926_REPO with column babel_4926_COL 
+select babel_4926_COL into BABEL_4926_REPO from BABEL_4926;
+go
+-- Indexing over column babel_4926_COL
+create index IDX_REPRODUCTION on BABEL_4926_REPO(babel_4926_COL);
+go
+-- output is the lowercase name,original_name
+select attname,attoptions from pg_attribute where attname like '%babel_4926%' and attrelid = (select oid from pg_class where relname = 'babel_4926_repo') order by attname asc;
```

Comment:
Not fancy enough ;-), let's come up with a query having join between pg_attribute and pg_class. Moreover, we don't want full `attoptions` list here, instead the query can be modified to just print the `bbf_original_name` element of that list.

[BONUS] Let's see if we can use `information_schema.columns` view instead for this purpose in which case you can ignore the suggestion above.